tests/Backend/CommonPart1/BucketsTest.php

```diff
@@ -640,6 +642,51 @@ public function testBucketOwner(string $devBranchType, string $userRole): void
         $bucket = $this->_testClient->getBucket($bucketId);
         $this->assertEquals($ownerId, $bucket['owner']['id']);
         $this->assertEquals($ownerName, $bucket['owner']['name']);
+        $this->assertNotNull($bucket['lastShareChangeDate']);
+    }
+
+    /**
+     * @dataProvider provideComponentsClientTypeBasedOnSuite
+     */
+    public function testBucketDescriptionMetadata(): void
```

Comment:
asi by som rad videl testy kde udatujes napr displayName ale `lastShareChangeDate` sa neupdatne. 
Plus by som ties otestoval asi ze sa to nastavi pri share akciach ked budes uz upravovat toto.

Reply:
good point, dodělám

Reply:
done, i s aktualizovanou dokumentací